<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEditHTSDialog</class>
 <widget class="QDialog" name="AddEditHTSDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenHTS- Add</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Subject:</string>
   </property>
   <property name="buddy">
    <cstring>subjectlineEdit</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Day:</string>
   </property>
   <property name="buddy">
    <cstring>daylineEdit</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="daylineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Time:</string>
   </property>
   <property name="buddy">
    <cstring>timelineEdit</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="timelineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="subjectlineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Replicates:</string>
   </property>
   <property name="buddy">
    <cstring>replineEdit</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="replineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>201</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Mean hormone concentration:</string>
   </property>
   <property name="buddy">
    <cstring>mhclineEdit</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="mhclineEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>191</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>16</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>120</y>
     <width>201</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Standard error of the mean:</string>
   </property>
   <property name="buddy">
    <cstring>semlineEdit</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="semlineEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>191</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddEditHTSDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddEditHTSDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
